### ISSUE TITLE
Include a countdown clock like they do on the news for big events

### ISSUE THREAD

**Issue by brennanMKE** [gravicle/sf-ios#14]
It would be a simple image view which shows a representation of a countdown to the next event. It would only show less than 48 hours before the displayed event as a way to indicate that the user is looking at the next event for this week.

I could implement this simple feature. I can build it as standalone project and make it accessible via CocoaPods and Carthage.

**Comment by brennanMKE** [comment]
A 48 hour countdown similar to these could work well. The ring could simply be a countdown for the number of hours to the next coffee meetup. It could change colors when it goes from days to hours to minutes. And these colors could be coffee themed.

<img width="644" alt="screenshot" src="https://user-images.githubusercontent.com/277419/30292077-9a49a760-96e9-11e7-90e9-4f2e1868fc45.png">

**Comment by rodericj** [comment]
I sort of like the simplicity of the current UX tbh. But if you really wanted an OSS you could check out https://github.com/danielamitay/DACircularProgress.

**Comment by gravicle** [comment]
In the detail view, the time string does kind of act as a count down. It’s not live, albeit. https://github.com/gravicle/sf-ios/blob/7e445a1e10d423adb0ec309f9f69a47651557faa/SF%20iOS/SF%20iOS/EventDetails/NSAttributedString%2BEventDetails.m#L15

**Comment by brennanMKE** [comment]
Yes, I'd want to stay simple. When I was looking at the detail view this morning it simply showed Wednesday without a date. Either the date could be added, which would require me to think of what the date is today, or it could show a circle that indicates how long till the event starts. I am thinking the circle would would have a ring to indicate visually how much time is left with text in the center.

* 2d
* 14h
* 18m

It could even be a faded gray to let it sort of blend into the detail view will still providing useful time context. With the time indicator showing days, hours and minutes it could be relevant all week long with a 7 day countdown. Beyond the 7 day period and after the meetup has started the indicator would not be visible.

**Comment by brennanMKE** [comment]
This is what I have so far. I put it together in PaintCode. The top image is the original drawing which is copied as a symbol with parameters for the other 3 drawings. These images would be scaled down to be placed on the detail screen. (see the screenshot below)

<img width="608" alt="countdown" src="https://user-images.githubusercontent.com/277419/30333231-51404e9a-9791-11e7-92a9-f2e349fc9899.png">

![mockup](https://user-images.githubusercontent.com/277419/30333719-826630c4-9792-11e7-9ba4-a5bb842cf8fa.png)
